SCClassLibrary/Common/Core/Object.sc

```diff
@@ -684,10 +678,10 @@ Object {
 		if (aSelector === '==', {
 			^false
 		},{
-			if (aSelector === '!=', {
-				^true
-			},{
-				BinaryOpFailureError(this, aSelector, [thing, adverb]).throw;
+		if (aSelector === '!=', {
```

Comment:
this reformatting seems wrong. Better leave it! Thanks!

Reply:
(maybe you could just undo all changes that are just formatting)

Reply:
I went back to develop to only include a single commit.